composer.json

```diff
@@ -1,14 +1,14 @@
 {
-    "name": "laravel-notification-channels/telegram",
-    "description": "Telegram Notifications Driver",
+    "name": "fastreserve/laravel-notification-channels/telegram",
```

Comment:
Please don't make changes to the composer file. Discard these changes and I'll accept the PR or create a new one.